apps/api/v1/next.config.js

```diff
@@ -47,6 +47,22 @@ const nextConfig = {
           source: "/v:version/:rest*",
           destination: "/api/v:version/:rest*",
         },
+        {
+          source: "/api/v2",
+          destination: `${process.env.NEXT_PUBLIC_API_V2_ROOT_URL}/health`,
+        },
+        {
+          source: "/api/v2/health",
+          destination: `${process.env.NEXT_PUBLIC_API_V2_ROOT_URL}/health`,
+        },
+        {
+          source: "/api/v2/docs/:path*",
```

Comment:
Idk if we need ":path*" for "docs", because either it is "/docs" or "/docs-yaml" returning docs in yaml format, so i would have "api/v2/docs" source and another object with "api/v2/docs-yaml" source.

Reply:
we need the path because of the scripts being downloaded for swagger ui